tests/metrics/test_chota.py

```diff
@@ -0,0 +1,54 @@
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+from traccuracy import run_metrics
+from traccuracy.loaders import load_ctc_data
+from traccuracy.matchers import CTCMatcher
+from traccuracy.metrics import CHOTAMetric, CTCMetrics
+
+
+def test_replicating_ctc_metrics_test() -> None:
+    project_root = Path(__file__).parent.parent.parent
+
+    gt_path = project_root / "downloads/Fluo-N2DL-HeLa/01_GT/TRA"
+    if not gt_path.exists():
+        pytest.skip(f"GT path not found at {gt_path}")
+
+    res_path = project_root / "examples/sample-data/Fluo-N2DL-HeLa/01_RES"
+    if not res_path.exists():
+        pytest.skip(f"RES path not found at {res_path}")
+
+    input_graph = load_ctc_data(str(res_path), run_checks=False)
+    gt_graph = load_ctc_data(str(gt_path), run_checks=False)
+
+    matcher = CTCMatcher()
+    metrics_func = [CTCMetrics(), CHOTAMetric()]
```

Comment:
After you have added the call for annotating ctc errors, let's remove running CTCMetrics here.
```suggestion
    metrics_func = [CHOTAMetric()]
```